src/Fields/Taxonomy.php

```diff
@@ -44,23 +44,23 @@ public function appearance(string $fieldType): self
         return $this;
     }
 
-    public function createTerms(): self
+    public function addTerm(bool $addTerm = true): self
```

Comment:
Using addTerm (singular) to match ACF https://github.com/AdvancedCustomFields/acf/blob/37fcf7204f034c0dcdf8240ffa6d802a238431dc/includes/fields/class-acf-field-taxonomy.php#L788